### ISSUE TITLE
Plots not being shown in the colab

### ISSUE THREAD

**Issue by mugheesahmad** [googlecolab/colabtools#1067]
- Describe the current behavior: The plots are not being shown in the Colab when I am running a python file.

- Describe the expected behavior: plot should be shown

- The web browser you are using (Chrome, Firefox, Safari, etc.): Chrome

- Link to self-contained notebook that reproduces this issue: [Link](https://colab.research.google.com/drive/1OUu6NjrxpGUiytEk2bkzNRODxnnienh-)

**Comment by colaboratory-team** [comment]
When you run 
```
!python scripy.py
```
the script is executed in a sub-shell that is not connected to the notebook kernel, and so there is no way to have resulting charts display in the notebook.

Instead you can use
```
%run script.py
```
to run your script within the current notebook session, and then plots will display inline.

**Comment by abduldattijo** [comment]
my plot is not showing on gooogle colab after running this code

def plot_history(history):
  """
    Plotting training and validation learning curves.

    Args:
      history: model history with all the metric measures
  """
  fig, (ax1, ax2) = plt.subplots(2)

  fig.set_size_inches(18.5, 10.5)

  # Plot loss
  ax1.set_title('Loss')
  ax1.plot(history.history['loss'], label = 'train')
  ax1.plot(history.history['val_loss'], label = 'test')
  ax1.set_ylabel('Loss')

  # Determine upper bound of y-axis
  max_loss = max(history.history['loss'] + history.history['val_loss'])

  ax1.set_ylim([0, np.ceil(max_loss)])
  ax1.set_xlabel('Epoch')
  ax1.legend(['Train', 'Validation'])

  # Plot accuracy
  ax2.set_title('Accuracy')
  ax2.plot(history.history['accuracy'],  label = 'train')
  ax2.plot(history.history['val_accuracy'], label = 'test')
  ax2.set_ylabel('Accuracy')
  ax2.set_ylim([0, 1])
  ax2.set_xlabel('Epoch')
  ax2.legend(['Train', 'Validation'])

  plt.show()

plot_history(history)


the cell runs, but shows nothing
